node_modules/gh-orgunit/tests/util.js

```diff
@@ -896,3 +896,26 @@ var createSimpleTree = module.exports.createSimpleTree = function(client, callba
         });
     });
 };
+
+/**
+ * Generate a set of organisational units
+ *
+ * @param  {Client}         client                  The client who will create the organisational units
+ * @param  {Number}         nrOfOrgUnits            The number of organisational units that should be created
+ * @param  {Function}       callback                Invoked when all the organisational units have been created
+ * @param  {OrgUnit}        callback.orgUnit1       The first organisational unit
+ * @param  {OrgUnit}        callback.orgUnit2       The second organisational unit
+ */
+var generateOrgUnits = module.exports.generateOrgUnits = function(client, nrOfOrgUnits, callback) {
+    var createdOrgUnits = [];
+    var done = _.after(nrOfOrgUnits, function() {
+        return callback.apply(callback, createdOrgUnits);
```

Comment:
I don't think this needs to change, but I learned not too long ago that `callback.apply(null, createdOrgUnits);` would be appropriate, as `callback` would be expected to be a function with no `this`.